benchmarks/encoder/config.yaml

```diff
@@ -21,3 +22,8 @@ one-metric:
 many-metrics:
   <<: *base_variant
   nmetrics: 48
+dd-origin:
+  <<: *base_variant
+  ntags: 10
+  ltags: 16
+  dd_origin: true
```

Comment:
we probably want one with no tags/metrics to show the overhead of just adding metrics/meta (e.g. `one-trace-dd-origin` to reflect the base variant.

what do you think?